compute_tools/src/extensions.rs

```diff
@@ -0,0 +1,103 @@
+use compute_api::responses::Extension;
+use compute_api::responses::ExtenstionsList;
+use url::Url;
+use std::collections::HashMap;
+
+// TODO limit the number of databases we query
+// TODO write proper version comparison
+//
+// TODO call it from somewhere, i.e. from ControlPlane after StartCompute (?)
+// and save it in some per-endpoint stat table in cplane
+
+
+use postgres::{Client, NoTls};
+use tokio::task;
+use anyhow::Result;
+use tracing::info;
+
+
+pub fn list_dbs(client: &mut Client) -> Result<Vec<String>> {
```

Comment:
>Yet, you also need to filter out dbs with not allowed connections

this is already part of the query

I added limit 50, which should be enough for most of the projects.
On a second thought, this code is not on the critical path, so I think it will be ok to query all databases.